core/templates/base.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="scroll-smooth">
```

Comment:
I would prefer to have this in the base.css file. We already have some specifications for the 'body' tag, we can just add it there. 

But don't worry about it in this PR, we will have to add these kinda general all purpose styling to base.css during style fixing. I have already created a for that and this smooth scroll is in the list :) 

For now you can remove it from here.